### PR TITLE
feat: Html converter package

### PR DESCRIPTION
**Pull request by freshtechtips**
- *Feature #1100*

Feel free to [vote](https://github.com/singerdmx/flutter-quill/discussions/1524) the package name

### CODE REVIEW

File: scripts/enable_local_dev.sh [hunk 1]
```diff
@@ -19,4 +19,9 @@ cp flutter_quill_test/pubspec_overrides.yaml.g flutter_quill_test/pubspec_overri
 
 echo ""
 
+echo "Enable local development for all the other packages..."
+cp packages/flutter_quill_html/pubspec_overrides.yaml.g packages/flutter_quill_html/pubspec_overrides.yaml
```
Comment (singerdmx) [inline scripts/enable_local_dev.sh:23]: is `.g` a typo?

**Review by singerdmx — APPROVED** [review]

File: scripts/enable_local_dev.sh (hunk 1 above)
Comment (freshtechtips) [reply scripts/enable_local_dev.sh:23]: No, it's not, I recently add a way to enable and disable local development

Comment (freshtechtips) [reply scripts/enable_local_dev.sh:23]: But I think it's a good idea to rename it with `.disabled` instead of `.g` to make it clear, let's do that next

**Comment by singerdmx** [comment]
Are you sure we want to introduce md here?

**Comment by singerdmx** [comment]
It would be nice only having logic for html and delta without use of md

**Comment by freshtechtips** [comment]
> Are you sure we want to introduce md here?

Not sure what you mean, I already added the README.md of the package

**Comment by singerdmx** [comment]
By md I mean markdown

**Comment by freshtechtips** [comment]
> It would be nice only having logic for html and delta without use of md

Yes, they can easily without the `README.md` I also updated the `example`

It only have option to convert the delta to HTML, then convert the HTML back to quill delta and load it, which can emulate the process faster

**Comment by singerdmx** [comment]
Is it like delta <-> mark down <-> html

**Comment by freshtechtips** [comment]
> By md I mean markdown

Currently, there is no built-in way to convert HTML to Delta without building it from scratch, so I use this solution as a workaround, it's from a contributor in GitHub

Of course we will update this package and do this from scratch or do it in better way but for now I think we should focus on `flutter_quill` and `flutter_quill_extensions`

**Comment by singerdmx** [comment]
Ok it does feel hacky but sure as long as it works

**Comment by freshtechtips** [comment]
> hacky

The logic for converting Delta to HTML is fine

The logic for converting the HTML back to Delta is not ready and we will change it in the future

**Pull request merged**